placement-rules-in-sql.md

```diff
@@ -173,6 +173,7 @@ The following known limitations exist in the experimental release of Placement R
 
 * Dumpling does not support dumping placement policies. See [issue #29371](https://github.com/pingcap/tidb/issues/29371).
 * TiDB tools, including Backup & Restore (BR), TiCDC, TiDB Lightning, and TiDB Data Migration (DM), do not yet support placement rules.
+* Using BR to restore a database or table with placement options will fail. See: [issue #31455](https://github.com/pingcap/tidb/issues/31455).
```

Comment:
Can you remove BR from the line above which says it is not supported? Otherwise the mention of BR is duplicated.

Reply:
Done